ipam/ring/ring.go

```diff
@@ -19,8 +19,14 @@ import (
 	"github.com/weaveworks/weave/router"
 )
 
+const MaxClockSkew int64 = int64(time.Hour)
+
+// Hook for replacing for testing
+var Now = func() int64 { return time.Now().Unix() }
```

Comment:
This could be non-exported, given current usage.  Or, if expected to be called in a test crossing Allocator and Ring, perhaps both should share the same time source?